src/xpanse-api/generated/apis/ServiceApi.ts

```diff
@@ -96,7 +96,7 @@ export class ServiceApiRequestFactory extends BaseAPIRequestFactory {
         }
 
         // Path Params
-        const localVarPath = '/xpanse/service/openapi/{id}'.replace('{' + 'id' + '}', encodeURIComponent(String(id)));
+        const localVarPath = '/xpanse/register/openapi/{id}'.replace('{' + 'id' + '}', encodeURIComponent(String(id)));
```

Comment:
this file must be auto-generated. No manual changes must be done.

Reply:
ok,I'm going to regenerate these files

Reply:
I discussed with xinsui just now, he is modifying the interface and entity, after he finishes modifying, I will regenerate these files